cEP-0022.md

```diff
@@ -0,0 +1,160 @@
+# Quickstart Green Mode
+
+| Metadata |                                               |
+| -------- | --------------------------------------------- |
+| cEP      | 22                                            |
+| Version  | 1.0                                           |
+| Title    | Quickstart Green Mode                         |
+| Authors  | Ishan Srivastava <mailto:ishan.srt@gmail.com> |
+| Status   | Implementation Due                            |
+| Type     | Feature                                       |
+
+# Abstract
+
+This document describes what is the **Green Mode** for `coala-quickstart` and how
+will this be implemented.
+
+This cEP describes new type of config files which will be generated by
+`coala-quickstart`, how will they be different from the ones currently being
+created, how is this helpful and the action-plan for adding this feature.
+
+# Introduction
+
+Currently `coala-quickstart` generates config files which point out too many
+errors in the code-base of the project. It also asks too many questions
+while generating the config file. The config files are not that specific to
+the project and the errors pointed out by coala when operated on the config
+files produced are generally undesirable for the project owners. 
```

Comment:
errors pointed out by coala when operated on the config files produced are generally undesirable for the project owners.

This seems ambiguous. Can you change it to something like:

errors  pointed out by coala using generated config files are generally undesirable for the project owners.